.editorconfig

```diff
@@ -7,3 +7,7 @@ indent_style = space
 indent_size = 2
 charset = utf-8
 trim_trailing_whitespace = true
+
+[*.out] # make editor neutral to .out files
```

Comment:
We have to use newlines at the end of files because of https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

I think there is no problem with it. We can just add newline in the test files or test cases

Reply:
I had a problem with it, test was failing